news/2024/2024-09-17-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Performance Points & Star Rating Updates
+date: 2024-09-17 12:00 +0000
+---
+
+The time for change is finally here!
+
+![](/wiki/shared/news/banners/star-rating.jpg)
+
+There are changes to all game modes this time around in an effort to improve the difficulty and performance calculations. This news post will display what has changed in a format that should be understandable to you, the player!
+
+If you find yourself scratching your head when reading, consider consulting the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) wiki articles to gain a greater understanding of the topics.
+
+## Release Schedule
+
+<!-- todo -->
```

Comment:
I've left this empty as I'm unsure what it will look like this time around. Likely requires input from @peppy.